week2/2418_sort_ppl.py

```diff
@@ -0,0 +1,8 @@
+# https://leetcode.com/problems/sort-the-people/
+from typing import List
+
+
+class Solution:
+    def sort_people(self, names: List[str], heights: List[int]) -> List[str]:
+        person_dict = {heights[i]: names[i] for i in range(len(names))}
+        return [v for k, v in sorted(person_dict.items(), reverse=True)]
```

Comment:
(Optional) This is kinda against what I said in the comment of `nth_fib`, but If you care about memory consumption, you can use an array of tuples instead. dictionary consumes more due to it being a hash map. Then, use a `lambda` function to indicate the order. via `sorted(keys =lamda...`